BukkyOmo/Questioner
# #163750634 implement fetch API to get all upcoming meetups

#### What does this PR do?
This PR:
* Implement Fetching API for get upcoming meetups endpoint.

#### Description of Task to be completed
* Create a js file for upcoming meetups.
* Include the get all meetups page heroku page url.
* Create variables for each field that is required.
* Create an event listener for the button to be clicked. 
* Create your fetch which should include method of http request, body and the headers.

#### How should this be manually tested?
* Clone this repository
* Check out to <ft-implement-fetch-API-to-get-all-upcoming-meetups-#163750634> branch
* Check upcomingmeetups.html page via browser.

#### Any background context you want to provide?
No

#### What are the relevant pivotal tracker stories? (if applicable)
#163750634
